app-catalog/src/api/charts.tsx

```diff
@@ -1,4 +1,18 @@
-import { PAGE_OFFSET_COUNT_FOR_CHARTS } from '../components/charts/List';
+import { request } from '@kinvolk/headlamp-plugin/lib/ApiProxy';
+import {
+  COMMUNITY_REPO,
+  CUSTOM_CHART_VALUES_PREFIX,
+  PAGE_OFFSET_COUNT_FOR_CHARTS,
+  VANILLA_HELM_REPO,
+} from '../components/charts/List';
+import { yamlToJSON } from '../helpers';
+import { isElectron } from '../index';
+
+const SERVICE_PROXY = '/serviceproxy';
```

Comment:
What is /serviceproxy?

Reply:
Answering my own question... it's provided in this PR: https://github.com/kubernetes-sigs/headlamp/pull/3532

Reply:
It's this PR https://github.com/kubernetes-sigs/headlamp/pull/3532